src/pages/authentication/building-todo-app.md

```diff
@@ -95,7 +94,7 @@ export const Signin = () => {
 
 This component imports the [React hook](https://reactjs.org/docs/hooks-overview.html)
 [`useConnect`](https://github.com/blockstack/ux/blob/master/packages/connect/src/react/hooks/use-connect.ts)
-from the Stacks Connect library.
+from the Stacks Auth library.
```

Comment:
Might we want to refer to authentication here as "the Stacks authentication library", avoiding the capitalization and abbreviation "Auth", which could be interpreted as a specific product brand (e.g. "Stacks Auth")?

Reply:
fixed